README.md

```diff
@@ -3,7 +3,15 @@
 [![Waffle.io - Columns and their card count](https://badge.waffle.io/deislabs/duffle.svg?columns=In%20Progress,Needs%20Review,Done)](https://waffle.io/deislabs/duffle)
 
 
-Duffle is a command line tool that allows you to install and manage CNAB bundles. To learn more about about CNAB and duffle, check out the [docs](docs/README.md).
+Duffle is the reference implementation of the [CNAB specification][cnab]. It
+provides a comprehensive mapping of _all_ features of the specification, serving
+both as a tool to install and manage bundles, and author bundles at a low level.
+
+The community has created implementations of the CNAB spec with
+[opinionated takes on authoring bundles][cnab-tools]. Some even use Duffle's
+[libraries][cnab-go] to handle the CNAB implementation. If you want to make your own CNAB tooling, that is a great place to start!
```

Comment:
Shouldn't the link `libraries` point to the libraries list on cnab.io, as there are multiple libraries (rust, python, .net) other than go?

Reply:
Oops! Yes, thank you good catch!